src/styles/index.scss

```diff
@@ -1,3 +1,144 @@
+$mainColor: #060b35;
+$secondaryColor: #616070;
+$buttonBlueColor: #00acdc;
+$buttonWhiteColor: white;
+$fontFamily: "Roboto", "Helvetica", sans-serif;
```

Comment:
you can create a separate variables.scss file and store your variables there. (just import this file into `index.scss` using `@import`)